created base html and css

kalebsutherland opened the pull request:
I'm submitting my first pull request for Skillcrush 105!

redrambles commented:
Hey @kalebsutherland!

This is great - the only thing we are missing is a link to your advice page from the main index.html file, so that folks can find your words of wisdom, once your code is merged in. If you're not sure how to do this, feel free to take a look at other students links for reference. :)

Once you add your link, add and commit your changes and then push your branch up once more, which will automatically update your pull request. Then we should be good to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

kalebsutherland commented:
Hi @redrambles! Thanks for flagging this. I added my link to the main index.html file. I *think* I successfully pushed it and updated the pull request. Let me know if I did anything wrong here. Thanks!

acodeinprogress commented:
Hey @kalebsutherland,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
